root/types.js

```diff
@@ -421,6 +421,7 @@ declare type PlaceT = {|
   ...CoreEntityRoleT,
```

Comment:
You also need to add `...DatePeriodRoleT,` after this line, and a line for `area:  AreaT | null,`.

To check the types locally and see if everything's fine, run `./node_modules/.bin/flow`